index.css

```diff
@@ -628,6 +650,191 @@ a:hover {
   margin-bottom: 2%;
 }
 
+.modal-container {
+  border-radius: 10px;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  position: fixed;
+  z-index: 99;
+  transition: all 0.1s ease;
+}
+
+.hidden {
+  display: none;
+}
+
+.box-container {
+  border-radius: 10px;
+  height: 1000px;
+  background-color: #fff;
+  transition: all 0.1s ease;
+}
+
+.modal-desktop-image {
+  width: 90%;
+  height: 260px;
+  margin-top: 5%;
+  margin-left: 1%;
+  margin-right: 1%;
+  margin-bottom: 5%;
+  border-radius: 5%;
+}
+
+.modal-project-title {
+  color: #172b4d;
+  margin-left: 5%;
+  margin-top: -15px;
+  line-height: 35px;
+  height: 82px;
+  font-weight: 700;
+  font-size: 30px;
+  width: 100%;
+  text-align: left;
+  margin-bottom: 20px;
+}
+
+.modal-content-btn {
+  text-align: right;
+  margin-left: 45%;
+  padding: 1%;
+  border-radius: 3px;
+  filter: contrast(150%);
+  display: none;
+}
+
+.modal-cancel-icon {
+  display: none;
+}
+
+.modal-cancel-container.hidden {
+  display: none;
+}
+
+.modal-cancel-mobile-icon {
+  display: block;
+  position: absolute;
+  top: 10%;
+  right: 11%;
+  cursor: pointer;
+}
+
+.modal-content-btn-btm {
+  text-align: center;
+  margin: 2px;
+  padding: 1%;
+  border-radius: 3px;
+  filter: contrast(150%);
+  display: block;
+}
+
+.modal-See-Source-btm {
+  background: #36b37f;
+  border-radius: 5px;
+  color: white;
+  padding: 12px 12px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 12px;
+  margin: 10px 10px;
+  cursor: pointer;
+}
+
+.modal-See-Live-btm {
+  background: #36b37f;
+  border-radius: 5px;
+  color: white;
+  padding: 12px 12px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 12px;
+  margin: 10px 10px;
+  cursor: pointer;
+}
+
+.modal-icon-liveDemo {
+  margin-bottom: -5px;
+  width: 15px;
+}
+
+.modal-icon-github {
+  margin-bottom: -5px;
+  width: 15px;
+}
+
+.modal-content-button {
+  display: flex;
+  flex-direction: row;
+}
+
+.modal-content-next-button {
+  margin-left: 20%;
+  cursor: pointer;
```

Comment:
We can make use of utility class for this too

Reply:
Ok